www/pfSense-pkg-Lightsquid/files/usr/local/pkg/lightsquid.inc

```diff
@@ -328,12 +335,16 @@ ssl.honor-cipher-order	= "enable"
 ssl.cipher-list		= "AES128+EECDH:AES256+EECDH:AES128+EDH:AES256+EDH:AES128-SHA:AES256-SHA:!aNULL:!eNULL:!DSS"
 EOF;
 
-		// Configure IPv4 and IPv6 sockets
+		/*
+		 * IPv4 HTTPS bind must NOT be a socket; otherwise the retarded
+		 * lighttpd thing yet again binds to 80/HTTP no matter what.
+		 * You enable SSL-only IPv4 by not configuring bind for IPv4 at all.
+		 * IPv6 SSL must be configured as a socket
+		 */
 		$lighttpd_ls_conf .= <<<EOF
 
```

Comment:
That block absolutely MUST be ther exactly as noted in the comments. Once for global scope and for each and every socket. I've damaged two keyboards while messing with this POS, trust me. Omit that block and the socket will be HTTP and not HTTPS.